src/olympia/amo/models.py

```diff
@@ -659,3 +659,16 @@ class GroupConcat(models.Aggregate):
 class Inet6Ntoa(Func):
     function = 'INET6_NTOA'
     output_field = CharField()
+
+
+class Metric(models.Model):
```

Comment:
I'm really not sure why we need a new model for this. It seems we are keeping track of counts so we can:
- use grafana counters
- logs
- google analytics (maybe not server side but you get the idea)

Models tend to expand and harden so if we can avoid adding one, I would.

Reply:
If this was just for engineering grafana or logs would be fine, but it's for Operations/Product, and their tool of choice is Redash. Notably because it allows correlation with the rest of the data available there

Reply:
Does redash not have access to other data sources? That's surprising to me but TIL. I want to open a slack thread about this but won't stop the PR for it. Just want to make sure we draw attention to this as it isn't great that this is the only way to get this data..

Reply:
Nit if we do indeed go this route. Metric is a vague name, and this should not be a vague model. I don't think we should open flood gates for adding all kinds of metrics and such... feels like the creation of `ActivityLog` if I can imagine how that went. We should avoid like plague and make this a hyper-specific model that resists being used for other purposes `ReviewQueueOptimizedCount` feels specific enough. It should also be in the review app so as not to be considered a general purpose thing.

Should other purposes arise we should double down on finding a solution in the place where it belongs, which is in redash or big query or some tool made for collecting metrics on existing data not our production database.

Reply:
Redash does have access to other sources but they either expire (statsd pings) or are not more practical than our database (bigquery).

Reply:
>  not more practical than our database

What's the argument for that statement? From where I'm standing it sounds like this is literally the use case for big query.

Reply:
AMO is not currently set up to write to BigQuery - at the moment it only uses it to read stats from, which are generated outside of AMO completely. And most of the data we'd want to correlate these metrics with exist on AMO side (activity log etc).

Reply:
That's probably something we should look into.. I'm not sure it really matters where other data we want to correlate to is, that's the benefit of a "data lake" tool like redash. But fine for now then. Wdyt about the specificication of the model then?

Reply:
I had missed your comment about the model being too vague. That's fair, I was worried we'd immediately need more data, which is why I wanted to be quite generic from the beginning (though I did make `value` an integer in an attempt to make it too generic and end up like `ActivityLog`). I don't mind making the name more specific to force us to use it correctly, I'll think about it a bit more.

Reply:
1ff5be491c57e4c5c09fc52b0707b18e289ae4bc renames and moves the model & the cron